src/chains/zetachain/withdraw.ts

```
@@ -0,0 +1,102 @@
+
+import {
+  GatewayContract,
+  RevertOptions,
+  TxOptions,
+  ZRC20Contract,
+} from "../../../types/contracts.types";
+import {
+  zetachainWithdrawParamsSchema,
+  zetachainOptionsSchema,
+} from "../../schemas/zetachain";
+import { toHexString } from "../../../utils/toHexString";
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)